antora-playbook.yml

```diff
@@ -78,7 +78,7 @@ content:
     branches: [release/2.8, release/2.7, release/2.6, release/2.5, release/2.1, release/2.0, release/1.5]
   - url: https://github.com/couchbase/docs-service-broker
     branches: [1.0.x]
```

Comment:
```suggestion
    branches: [1.1.x, 1.0.x]
```

Reply:
Scope creep Eric, the request says Operator :wink:

Reply:
It's spelled C-L-O-U-D-N-A-T-I-V-E-D-A-T-A-B-A-S-E